src/layer/Tooltip.js

```diff
@@ -387,15 +387,19 @@ Layer.include({
 	},
 
 	_addFocusListenersOnLayer: function (layer) {
-		DomEvent.on(layer.getElement(), 'focus', function () {
-			this._tooltip._source = layer;
-			this.openTooltip();
-		}, this);
-		DomEvent.on(layer.getElement(), 'blur', this.closeTooltip, this);
+		if (layer.getElement()) {
+			DomEvent.on(layer.getElement(), 'focus', function () {
```

Comment:
Nit: while we're at it, let's avoid repeated calls to the same thing. E.g. do `var el = layer.getElement();` first.

Reply:
Changed